torch/csrc/distributed/c10d/symm_mem/NCCLSymmetricMemory.cu

```diff
@@ -2,11 +2,13 @@
 #include <nccl.h>
 #include <torch/csrc/cuda/nccl.h>
 
-#if NCCL_VERSION_CODE >= NCCL_VERSION(2, 27, 1)
+#if NCCL_VERSION_CODE >= NCCL_VERSION(2, 28, 9)
```

Comment:
Usually features are the same within a minor version (2.28), not sensitive to patch version (2.28.9). We can use (2, 28, 0) here.